packages/codemods/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "@fluentui/codemods",
   "version": "8.4.1",
   "description": "Tool enabling easy upgrades to new Fluent UI versions",
-  "main": "lib/index.js",
-  "typings": "lib/index.d.ts",
+  "main": "lib-commonjs/index.js",
```

Comment:
why are ther some packages that change entrypoints to common-js but don't update the build command to use the new flags ?

Reply:
`commonjsOnly` (was putting `cjs` output in `lib` folder) task was removed thus build assets are produced in unified folders

esm -> lib
cjs -> lib-commonjs
amd -> amd-commonjs

as can bee seen in the implementation and described in the comments, `build` works as before without need to refactor whole repo. next step is to start using `--module` for v9 packages. please see the Story that is referenced in this PR.

Reply:
that's fair, I would have thought since you were modifying the package.json files anyway might as well make the change to be more explicit

Reply:
> modifying the package.json files anyway might as well make the change to be more explicit

can you elaborate on how could that look like ?

Reply:
so in codemodes/package.json couldn't you just change the `build` script to be `just-scripts build --module cjs` ?

Reply:
gotcha,

so I could but it wouldn't not the need of change in `main` field as this was using `commonJS` only task which transpiled cjs to `lib` (inconsistency). besides that I didn't want to touch v8 stuff with the new flag ATM.

hope it makes more sense now